nss-cohort-37/Workforce-Green-Monkeys
# Case create computer + employee

# Description
User can now assign employee to a computer upon creation,
Steps for testing -
create a computer assign an employee
check the computer data to confirm computer id number
check the employee data to confirm computer id matches the assigned employee's ComputerId
## Type of change
Please delete options that are not relevant.
- [x ] New feature (non-breaking change which adds functionality)
# Testing Instructions
Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
